src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -192,6 +192,134 @@
                             <RowDefinition Height="Auto" />
                             <RowDefinition Height="Auto" />
                         </Grid.RowDefinitions>
+                        <TextBlock x:Uid="ColorScheme_Black"

```

Comment:
oh, i see. you're NOT using ColorTableEntry...

Reply:
This invalidates my comments about the ColorTableEntry resource keys ...

Reply:
Yep! Will resolve those conversations